src/engraving/dom/tuplet.cpp

```diff
@@ -839,4 +840,24 @@ int Tuplet::computeTupletDenominator(int numerator, Fraction totalDuration)
     }
     return ratio.numerator();
 }
+
+EngravingItem* Tuplet::nextElement()
+{
+    ChordRest* firstElement = toChordRest(elements().front());
+    if (firstElement->type() == ElementType::CHORD) {
+        Chord* chord = toChord(firstElement);
+        return chord->firstGraceOrNote();
```

Comment:
This line causes the builds to [fail](https://github.com/musescore/MuseScore/actions/runs/10377706127/job/28732473002?pr=23082#step:12:1841) because the `Note` type is "incomplete". It has been forward declared but not defined at this point, so the compiler doesn't know that a `Note` is also an `EngravingItem`.

You need to add `#include "note.h"` to the top of this file. Always do this in the .cpp rather than the .h if you can.